specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/scheduledEvents.json

```diff
@@ -0,0 +1,175 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2023-11-02-preview"
+  },
+  "host": "management.azure.com",
```

Comment:
This doesn't seem right to me. The host is _not_ `management.azure.com` because this payload shape isn't served by any `management.azure.com` URL, right?

I'd give maybe the ARG host here...?

Reply:
As discussed with the ARN team, ARM will support such REST calls through rerouting the request to ARG. For now, it is not used, but needed for swagger semantics.